tests/helpers/config.js

```diff
@@ -66,4 +68,53 @@ exports.config = {
         delete browser.Cucumber_Test
         browser.pause(30)
     },
+
+    before() {
+        global.WDIO_SERVICE_TEST_IT_DURATION = 0
+        global.WDIO_SERVICE_TEST_IT_PASSES = 0
+
+        global.WDIO_SERVICE_TEST_HOOK_DURATION = 0
+        global.WDIO_SERVICE_TEST_HOOK_PASSES = 0
+    },
+    beforeTest(test) {
+        if (global.WDIO_SERVICE_TEST_IT_DURATION === 0) {
+            throw new Error('wdio beforeTest error: ' + test.title)
+        }
+    },
+    afterTest(test, context, { error, duration, passed }) {
+        let throwError = false
+        if (global.WDIO_SERVICE_TEST_IT_DURATION === 0) {
+            throwError = true
+        }
+
+        global.WDIO_SERVICE_TEST_IT_DURATION += duration
+        if (!error && passed === true && browser.pause(2) === undefined) {
+            global.WDIO_SERVICE_TEST_IT_PASSES++
+        }
+
+        if (throwError) {
+            throw new Error('wdio afterTest error: ' + test.title)
+        }
+    },
+    beforeHook(test) {
+        if (global.WDIO_SERVICE_TEST_HOOK_DURATION === 0) {
+            throw new Error('wdio beforeHook error: ' + test.title)
+        }
+    },
+    async afterHook(test, context, { error, duration, passed }) {
+        await sleep(20)
```

Comment:
needed?

Reply:
We aren't waiting for anything here, I just want to make sure that async command is executed properly. 
I think it would be better to replace it with browser.sleep, am I right?

Reply:
yes